src/redis/README.md

```diff
@@ -0,0 +1,28 @@
+# VIP Go Redis Helper Library
+
+## Initialization
+
+Instantiate a new Redis client:
+
+``` js
+const { redis } = require( '@automattic/vip-go' );
+const client = redis();
+```
+By default, this library logs to the `console`. If you prefer, you can set your own logger:
+
+```js
+const redis = redis( { logger: yourLogger } );
+```
+
+## Configuration
+
+Configuring a Redis client compatible with the VIP Go platform requires a `REDIS_MASTER` environment variable. This is the host and port of your Redis server, which might look something like this:
+`REDIS_MASTER = 123.456.789.123:3000`
+
+If you require a password, please set the `REDIS_PASSWORD` environment variable as well.
```

Comment:
Should we add a note here to mention that these vars are setup by default on VIP Go?

Reply:
Yes, sounds good. Added in https://github.com/Automattic/vip-go-node/pull/188/commits/d3da80dc46b037f080fa9301ab459ebbe8e9a3f6